public/static/js/distros.js

```diff
@@ -214,6 +223,47 @@ mciModule.controller('DistrosCtrl', function ($scope, $window, $http, $location,
       });
   };
 
+  $scope.updateSettingsList = function() {
+    // don't save if there aren't any populated fields
+    for (let [key, val] of Object.entries($scope.activeDistro.settings)) {
+        if (key !== "region" && val !== "" && val !== undefined) {
```

Comment:
I want to determine if there are any populated fields _besides_ region.

Reply:
Oops missed that. Would it be sufficient to check 1 field that's required (maybe AMI)? I think the current check may not work, since I believe binding text entry fields to properties on this will make them "" rather than undefined

Reply:
This checks both !== "" and also val !== undefined.
I don't want to check just one field that's required because users might be flipping between regions (maybe to compare values) without being done populating values.